### PR TITLE
Fe user profile

### PR DESCRIPTION
**Pull request by eeels22**
# Changes proposed

- Update `UserApi` with updateUserDescription function
- Code React component for `ProfileForm` with input for description and button.
- Existing `description` shows in the `textarea`
- Code React component for `ProfileCard` to display user info and ProfileForm 
- Code React component for `ProfilePage`

## Not in scope

- CSS/styling

## User story
`As a user of Cuddly, I want to fill up my user detail, so that I can display my information`

## Screenshots
`ProfileForm` shows existing description in the `textarea`
Profile form showing existing description

`ProfileCard` suggests that the user addd a `description` if it is null
Profile card prompts for description is it is null

### CODE REVIEW

File: frontend/src/pages/profile/ProfilePage.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// NPM Packages
+import React, { useEffect, useState } from "react";
+
+// Project files
+
+import ProfileCard from "../../components/profile/ProfileCard";
+import UserApi from "../../api/UserApi";
+
+export default function ProfilePage() {
+  // Lobal state
```
Comment (NeuralAlchemist) [inline frontend/src/pages/profile/ProfilePage.jsx]: just a typo, would you please correct it to Local?

**Review by NeuralAlchemist — APPROVED** [review]
everything looks good just a very tiny typo

File: frontend/src/pages/profile/ProfilePage.jsx (hunk 1 above)
Comment (eeels22) [reply frontend/src/pages/profile/ProfilePage.jsx]: Good spot, fixed!

**Review by janko101 — APPROVED** [review]
Good Job, En-Chi

**Pull request merged**